optuna/study.py

```diff
@@ -299,7 +299,7 @@ def trials_dataframe(self, include_internal_fields=False):
         histogram of objective values and to export trials as a CSV file. Note that DataFrames
         returned by :func:`~optuna.study.Study.trials_dataframe()` employ MultiIndex_, and columns
         have a hierarchical structure. Please refer to the example below to access DataFrame
-        elements.
+        elements. If there are no trials, an empty DataFrame_ is returned.
```

Comment:
Not really significant to this PR. I can revert this change if that's better.

Reply:
Nice addition 👍